unleash-yggdrasil/tests/grammar_prop_tests.rs

```diff
@@ -0,0 +1,12 @@
+use proptest::prelude::*;
+use unleash_yggdrasil::strategy_parsing::compile_rule;
+
+proptest! {
+    #[test]
+    fn test_compile_rule(input in any::<String>().prop_filter("Exclude strings with \\\", \", \\ and empty strings", |s| !s.is_empty() && !s.contains("\\\"") && !s.contains("\"") && !s.contains("\\"))) {
```

Comment:
We just strip out cases where we have our escaping characters. In most cases those will fail by design, in the cases where they're appropriate they're covered by concrete tests elsewhere